src/Compilers/Core/CodeAnalysisTest/EditAndContinue/EditAndContinueTest.cs

```diff
@@ -0,0 +1,182 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+extern alias PDB;
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Runtime.InteropServices;
+using Microsoft.CodeAnalysis.Emit;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using PDB::Roslyn.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.EditAndContinue.UnitTests
+{
+    internal abstract partial class EditAndContinueTest<TSelf>(Verification? verification = null) : IDisposable
```

Comment:
Consider having the primary ctor parameter name exactly match the field name so we can't accidentally capture the parameter.

Reply:
Doesn't the compiler report warning for accidental capture?